ashley/src/com/badlogic/ashley/signals/Signal.java

```diff
@@ -56,9 +56,8 @@ public void removeAllListeners () {
 	 * @param object The object to send off
 	 */
 	public void dispatch (T object) {
-		Object[] items = listeners.begin();
-		for (int i = 0, n = listeners.size; i < n; i++) {
-			Listener<T> listener = (Listener<T>)items[i];
+		final Listener<T>[] items = listeners.begin();
```

Comment:
Actually, this broke the build.

http://libgdx.badlogicgames.com:8080/job/ashley/103/console